internal/api/phone_test.go

```diff
@@ -258,3 +257,14 @@ func (ts *PhoneTestSuite) TestMissingSmsProviderConfig() {
 		}
 	}
 }
+
+func (ts *PhoneTestSuite) generateAccessTokenAndSession(ctx context.Context, u *models.User, authenticationMethod models.AuthenticationMethod) string {
```

Comment:
It would be great if this could be a global static method though I'm not sure if there's currently a way to do so

Reply:
yeah, not too sure if it's worth abstracting this into a function since we're only using it in one place in this file 

nit: if you opt to keep it, can we rename it to `generateAccessTokenString` since the function only returns the access token and not the session?

Reply:
Will drop it